__tests__/analyzer/schema-generator.js

```diff
@@ -0,0 +1,20 @@
+const { join } = require('path');
+const { generateSchema } = require('../../src/analyzer/schema-generator');
+const { FileSystemFileBrowserAdapter } = require('../../src/analyzer/file-browser');
+const emptyRepoPath = join(__dirname, '../data/empty-repo');
+const azimuthRepoPath = join(__dirname, '../data/azimuth');
+
+describe('generate schema', () => {
+    test('empty repo', async () => {
+        const fileBrowserAdapter = new FileSystemFileBrowserAdapter({ dirPath: emptyRepoPath });
+        const schemaGeneratorResult = await generateSchema({ ssgMatchResult: null, fileBrowserAdapter });
+        expect(schemaGeneratorResult).toBeNull();
```

Comment:
shouldn't be null, needs to be object with empty models, undefined dataDir and pagesDir